openvm/src/customize_exe.rs

```diff
@@ -105,14 +99,39 @@ pub fn customize(
 
     let opcodes_allowlist = airs.allow_list();
 
-    let labels = add_extra_targets(&exe.program, labels.clone());
-
-    let blocks = collect_basic_blocks(
+    let labels = add_extra_targets(
         &exe.program,
-        &labels,
-        &opcodes_allowlist,
-        &branch_opcodes_set(),
+        labels.clone(),
+        exe.program.pc_base,
+        exe.program.step,
+    );
+
+    let program = SymbolicProgram::new(
+        exe.program
+            .instructions_and_debug_infos
+            .iter()
+            .map(|o| o.as_ref().unwrap().0.clone())
+            .map(|instr| SymbolicInstructionStatement {
```

Comment:
Any reason to use usize here?

Reply:
For the opcode? It's the existing type for the opcode in `SymbolicInstructionStatement` before this PR